test/integration/app_test.exs

```diff
@@ -0,0 +1,67 @@
+defmodule Integration.AppTest do
```

Comment:
Even more so writing integration tests for `function_from_file/5` in docstrings makes even less sense as it's not possible to see the contents of the template files in the documentation.